.clang-format

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
```suggestion
# SPDX-License-Identifier: LGPL-2.1
```

maybe also add the copyright notice

Reply:
I can't legally change that. This is based in the Linux kernel file which uses GPL-2.0.

It will not be linked into firmwares using RIOT, so this should not be an issue for users who don't want to use GPL-2.0 code in their products.

Reply:
Ah, my bad. Then please add a reference to the original file.

Reply:
Done and squashed.